apps/api/src/helpers/purl_helpers.ts

```diff
@@ -7,23 +7,12 @@ import { PackageURL } from "packageurl-js";
 export const parsePurl = (purl: string) => {
     const parsedPurl = PackageURL.fromString(purl);
 
-    let qualifiers = undefined;
-
-    if (parsedPurl.qualifiers) {
-        for (const [key, value] of Object.entries(parsedPurl.qualifiers)) {
-            if (qualifiers) {
-                qualifiers += "&" + key + "=" + encodeURIComponent(value);
-            } else {
-                qualifiers = key + "=" + encodeURIComponent(value);
-            }
-        }
-    }
     return {
         type: parsedPurl.type,
         namespace: parsedPurl.namespace,
         name: parsedPurl.name,
         version: parsedPurl.version,
-        qualifiers: qualifiers,
+        qualifiers: purl.split("#")[0].split("?")[1],
```

Comment:
Does this work correct for purl without subpaths / qualifiers? I'm specifically looking at the index operations. Might they go out of bounds?

Reply:
If the character doesn't exist, the resulting array will have just one item, so the arr[1] will be undefined, which is an acceptable value for the qualifier.

Reply:
> I can add `?? undefined` in the end to make it more clear if needed

No, it's fine, I just wanted to double-check.